Promact.Oauth.Server/src/Promact.Oauth.Server/Controllers/UserController.cs

```diff
@@ -459,17 +559,25 @@ public IActionResult UserDetailById(string userId)
           * @apiSuccessExample {json} Success-Response:
           * HTTP/1.1 200 OK 
           * {
-          *     "description":"true"
+          *     "true"
+          * }
+          * @apiError InvalidApiRequestException , ArgumentNullException
+          * @apiErrorExample {json} Error-Response:
+          * HTTP/1.1 400 Bad Request
+          * {
+          *   "error": "InvalidApiRequestException"
+          *   "error": "ArgumentNullException"
           * }
           */
         [HttpGet]
-        [Route("reSendMail/{id}")]
-        public async Task<IActionResult> ReSendMail(string id)
+        [Route("email/{id}/send")]
+        [Authorize(Roles = "Admin")]
+        public async Task<IActionResult> ReSendMailAsync(string id)
         {
             try
             {
                 _logger.LogInformation("Resend Mail");
-                return Ok(await _userRepository.ReSendMail(id));
+                return Ok(await _userRepository.ReSendMailAsync(id));
             }
             catch (InvalidApiRequestException apiEx)
```

Comment:
I have removed this exception because it is not an exception that is thrown often